motr/motr-pub.api

```diff
@@ -460,3 +460,4 @@ m0_iem
 m0_console_printf
 m0_console_flush
 m0_error_printf
+print_xprt
```

Comment:
m0_net_xprt_printf?

Reply:
This is wrong.

Public API means, this API is callable by Motr application, linking against libmotr.so.
There is not need to export this as an public API.

Reply:
incorporated